scheduler/scheduler.go

```diff
@@ -187,6 +187,12 @@ func (s *scheduler) createTask(sch schedule.Schedule, wfMap *wmap.WorkflowMap, s
 
 	// Bind plugin content type selections in workflow
 	err = wf.BindPluginContentTypes(s.metricManager)
+	if err != nil {
```

Comment:
I had looked at this last week and wondered if it wouldn't be better to go ahead to the next block and capture if plugins are missing also. Return on either and/or.

The idea would be to return more of the failure reasons inclusive of binding problems and missing plugins.